core/common/boot/src/test/java/org/eclipse/edc/boot/system/ExtensionLoaderTest.java

```diff
@@ -98,6 +98,36 @@ void loadMonitor_whenNoMonitorExtension() {
         assertTrue(monitor instanceof ConsoleMonitor);
     }
 
+    @Test
+    void loadMonitor_programArgsSetConsoleMonitorLogLevel() {
+
+        var monitor = ExtensionLoader.loadMonitor(new ArrayList<>(), ConsoleMonitor.LEVEL_PROG_ARG + "=INFO");
+
+        var expectedMonitor = new ConsoleMonitor(ConsoleMonitor.Level.INFO, true);
+
+        assertEquals(monitor, expectedMonitor);
```

Comment:
please use assertjs `assertThat` kind of assertions

Reply:
[73f5085](https://github.com/eclipse-edc/Connector/pull/4476/commits/73f5085ddf516f1d8b29ec1c45e9c2c737ac0c90)